### ISSUE TITLE
Help is difficult to find, particularly using VoiceOver, and not available everywhere across the app

### ISSUE THREAD

**Issue by ctarda** [wordpress-mobile/WordPress-iOS#15001]
This issue has been reported by a user that navigates the app using VoiceOver.

Help is available within the "Me" screen. Which, in turn, means that it is only available within the "My Sites" tab, and when one site has been selected.

### Expected behavior
Help should be available everywhere. The user reporting the issue suggested either a new tab or a floating button.

### Actual behavior
Help is available only within the "Me" screen.

### Steps to reproduce the behavior
Navigate to Notifications or Reader, or the root of the My Site tab and notice how help is not available

##### Tested on [device], iOS [version], WPiOS [version]
Tested on iPhone 6s+, iOS 13.7, WPiOS 15.7, es_ES
Tested on iPhone 11 Pro Max, iOS 14.0.1, WPiOS 15.7, en_CA

**Comment by designsimply** [comment]
The 'me'/profile section was moved to the top of the My Site tap in 2019 as part of an Information Architecture Project.
<sub>(internal references: pauD4L-8W-p2 and pbArwn-2G-p2)</sub>

I found a related note in the internal documentation:

> The ‘me’/profile section will move to the top right of the My Site tab. The ‘Me’ section added one more option to decide between at the top level, but its not one of the top tasks. There were no usability issues moving it to the My Site tab.

@mattmiklic are there any improvements we should consider based on this accessibility feedback or can you speak to the decision with an explanation for us if you feel we should keep the information architecture as is?

**Comment by mattmiklic** [comment]
@osullivanchris can you take a look at this please? I'm not sure the suggestions of a new tab or FAB are the best choices but it'd be worthwhile to look at how some other apps are handling this, especially those that use a similar treatment for the personal settings in the top bar.

**Comment by osullivanchris** [comment]
The core of the problem is in the below quote from @ctarda 

> Help is available within the "Me" screen. Which, in turn, means that it is only available within the "My Sites" tab, and when one site has been selected.

The location of 'Me' is intended to be at the top level. However, because we currently still treat the site switcher as the top level of this tab, it means that help lives at a deeper level.

The intention is to change 'My Site' to be the top level destination. And 'Switch Site' to be a child of 'My Site' - accessed through a chevron or similar per Android. 

![IMG_20201002_125145](https://user-images.githubusercontent.com/50576199/94920694-aee50f80-04ae-11eb-9b8a-b83350529ef3.jpg)


In this case, the journey is 
App opens on My Site...Tap Profile icon -> Tap Help & Support. Much less hidden and Help is two taps from the main screen. 

So I agree with @mattmiklic  that a permanent tab or FAB would be an unnecessary overkill once we fix this. 1 tap away from the main screen to find it seems fine.

cc @frosty

**Comment by rezzap** [comment]
There was an internal report of the 'Me' profile section being difficult to find also here: p4a5px-2FS-p2 

> I was curious to find my profile to see what account I was logged in and all that and it took me surprisingly long time to find it.

> What I did:

    Opened the app.
    Clicked on Reader – didn’t see an icon at top right.
    Clicked notifications – also didn’t see a profile icon.
    Clicked My Sites – same.
    After some more clicking and tapping ended up on a specific site and only then I saw the profile icon. Yay!

> What I expected:

    To have had access to the profile a tiny bit sooner somehow.

iOS 14.2, iPhone 11 Pro, WP iOS 16.3

I'll also add a +1 from me for this one as I've always found this very difficult to access.

**Comment by designsimply** [comment]
I see two distinct problem reports here:

1. Help should be available everywhere.
1. User profile is difficult to find.

I found two potential solutions proposed:

1. Make 'My Site' to be the top level destination. (proposed in a [comment above](https://github.com/wordpress-mobile/WordPress-iOS/issues/15001#issuecomment-702692997))
1. Make the user icon visible on all top-level screens. (mentioned at internal reference p4a5px-2FS-p2#comment-11594)

I opened a new issue for the second problem at https://github.com/wordpress-mobile/WordPress-iOS/issues/15493 because I think the request there is more clear, and therefore more actually, and still leaves space for the problem of help being hard to find for consideration in this issue.

**Comment by frosty** [comment]
Just a note that we're actively working on item 1 at the moment – in a future update (likely in January), My Site will be the top level of the My Site tab.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
